core/domain/email_manager.py

```diff
@@ -333,6 +333,37 @@ def _send_bulk_mail_in_transaction(instance_id=None):
         _send_bulk_mail_in_transaction, instance_id)
 
 
+def send_email(
+        recipient_id, sender_id, intent, email_subject, email_html_body,
+        sender_email, bcc_admin=False, sender_name=None, reply_to_id=None):
+    """Public wrapper for the _send_email method.
```

Comment:
This isn't a pattern that we're going for. Take a look at this comment: https://github.com/oppia/oppia/pull/7703#discussion_r329400201 and this one: https://github.com/oppia/oppia/pull/7703#discussion_r331582495

Reply:
@kevinlee12 I think we have to make `_send_email` method public as I am not able to find a way to test the code in the private method through a public method.

Reply:
We don't want to make methods public if they don't need to be. Take a look at Sean's comment on the suggested approaches.